exchange/utils.go

```diff
@@ -292,6 +292,7 @@ func buildRequestExtForBidder(bidder string, requestExt json.RawMessage, request
 		Debug:               requestExtParsed.Prebid.Debug,
 		CurrencyConversions: requestExtParsed.Prebid.CurrencyConversions,
 		BidderParams:        bidderParams,
+		Server:              requestExtParsed.Prebid.Server,
```

Comment:
Good on adding the server info here. Please update the tests in `TestBuildRequestExtForBidder` to include test data for the new server field. Is this information being set? That is the job of PBS core logic and is not left up to the bidders.